src/source/Main.scala

```diff
@@ -282,6 +285,15 @@ object Main {
     } else {
       None
     }
+    val objcSwiftBridgingHeaderWriter = if (objcSwiftBridgingHeaderOut.isDefined) {
```

Comment:
Because the generators work differently.
The ObjC generator writes the files one but one, and only start the next one, when the current is finished.
The SwiftBridgingHeaderGenerator capture the ObjC file names while they are being generated, and writes them to the BridgingHeader.
So the SwiftBridgingHeaderGenerator is writing the BridgingHeader since djinni start generating ObjC files, until it finishes.